.github/workflows/release-please.yml

```diff
@@ -46,3 +46,36 @@ jobs:
           docker tag topo-imagery ghcr.io/linz/topo-imagery:${GIT_VERSION}
 
           docker push --all-tags ghcr.io/linz/topo-imagery
+
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v2
+        with:
+          aws-region: ap-southeast-2
+          mask-aws-account-id: true
+          role-to-assume: ${{ secrets.AWS_CI_ROLE }}
+
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+
+      - name: Tag and push docker image to Amazon ECR
+        env:
+          REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+          REPOSITORY: eks
+          IMAGE_TAG: ${{ github.sha }}
+        run: |
+          GIT_VERSION=$(git describe --tags --always --match 'v*')
```

Comment:
You could maybe have set these values as an environment variable from the publish container step (or an independent step), to avoid doing this again here. (https://docs.github.com/en/github-ae@latest/actions/using-workflows/workflow-commands-for-github-actions#example-of-writing-an-environment-variable-to-github_env)